nativelink-store/src/redis_store.rs

```
@@ -660,7 +660,7 @@ impl StoreDriver for RedisStore {
 
     fn register_remove_callback(
         self: Arc<Self>,
-        _callback: &Arc<Box<dyn RemoveItemCallback>>,
+        _callback: Arc<dyn RemoveItemCallback>,
```

Something is wrong here

---

What do you mean?  It's simply updating to the trait change.

---

It's worth noting that this should probably return an error in the same way that the gRPC store does because the Redis LTO does not notify on object removal.